.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: node_js
 node_js:
+  - "6"
   - "8"
-  - "7"
+  - "9"
```

Comment:
maybe `"stable"` instead of `"9"`? ..and only LTS versions define explicitly

Reply:
Good question. @JorgeBucaran what do you think?

Reply:
What do you folks make out of this?

<img width="885" alt="screen shot 2017-12-04 at 23 49 27" src="https://user-images.githubusercontent.com/56996/33558664-cec1f910-d94d-11e7-9f26-84a08db9fe98.png">

Reply:
https://github.com/travis-ci/docs-travis-ci-com/issues/362

Reply:
lots of options 😂